### PR TITLE
MM Ported Over Derived Table Instance series

### PR DESCRIPTION
**Pull request by jenmawe**
New derived table ported from LDP to MetaDB for series information in instance records

### CODE REVIEW

**Review by natalyapik — COMMENTED** [review]
runs and looks good!

**Comment by natalyapik** [comment]
- [v] Pull request has a complete description
- [v] Query runs without errors
- [v] Query output is correct
- [v] Query logic is clear and well documented
- [v] Query is readable and properly indented
- [v] Table and column names are in all-lowercase
- [v] Query begins with user documentation in comment lines
- [v] File name is listed in `runlist.txt` after dependencies
- [v] All columns have indexes
- [v] Table is vacuumed and analyzed

File: sql_metadb/derived_tables/instance_series.sql [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+CREATE TABLE instance_series AS 
+SELECT 
+	it.id AS instance_id,
+	it.hrid AS instance_hrid,
+	instseries.jsonb #>> '{}' AS series,
+    instseries.ordinality AS series_ordinality
+FROM 
+	folio_inventory.instance__t AS it
+	LEFT JOIN folio_inventory.instance AS inst ON inst.id::uuid = it.id::uuid
```
Comment (stdombek) [inline sql_metadb/derived_tables/instance_series.sql:12]: Both IDs are of type uuid. The type casting can be removed.

**Review by stdombek — APPROVED** [review]
- [X] Query runs
- [X] Data output makes sense
- [X] Data types look correct
- [X] Query logic seems correct
- [X] Column names make sense and are spelled correctly
- [X] Query syntax and style are appropriate (e.g., correct use of double quotes, lower case for data types)

For derived tables:
- [X] query has indexes on columns
- [X] vacuum analyze has been added to the bottom
- [X] table appears in runlist.txt at an appropriate point

Good work! Please check the type casting. I put approved because it's just a small thing.

**Review by natalyapik — APPROVED** [review]

**Pull request merged**